docs/source/tutorials/text.ipynb

```diff
@@ -106,9 +107,9 @@
    "source": [
     "# Package installation (hidden on docs.cleanlab.ai).\n",
     "# If running on Colab, may want to use GPU (select: Runtime > Change runtime type > Hardware accelerator > GPU)\n",
-    "# Package versions we used: tensorflow==2.9.1 scikeras==0.9.0 scikit-learn==1.1.3 tensorflow_datasets==4.5.2\n",
+    "# Package versions we used: tensorflow==2.9.1 scikit-learn==1.2.0 tensorflow_datasets==4.5.2\n",
```

Comment:
I don't think you removed scikeras from the !pip install command from the above cell